src/styles/ForecastSummaries.css

```diff
@@ -0,0 +1,4 @@
+.forecast-summaries {
+    display: flex;
+    justify-content: space-between;
+};
```

Comment:
You should use 2 spaces of indentation as you are in the other parts of the app:
```
.forecast-summaries {
  display: flex;
  justify-content: space-between;
};
```